server/src/environments/environment.ts

```diff
@@ -17,6 +17,7 @@ tags.push('containerV' + process.env.CONTAINER_VERSION, 'pkgV' + process.env.npm
 let envConfig = {
     ENV: environmentName,
     COVID19API_URL: process.env.COUNTRIESDATA_URL,
+    ALTERNATIVECOVID19API_URL: process.env.ALTERNATIVECOVID19API_URL || 'https://api.covid19api.com',
```

Comment:
Not in use right now. Since api was not in good shaper yesterdays evening.
But I decided to leave this